### PR TITLE
Remove ts-node from javascript examples

### PR DESCRIPTION
**Pull request by deadcoder0904**
I don't know why `ts-node` is required, maybe it stayed there while porting the examples but folks coming from vanilla JS won't know or wouldn't wanna know about `ts-node`.

This PR removes `ts-node` from `examples/javascript`.

### CODE REVIEW

File: examples/javascript/script/yarn.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (janpio) [inline examples/javascript/script/yarn.lock]: Intentional that you added this file?

Comment (deadcoder0904) [reply examples/javascript/script/yarn.lock]: yeah, why not? since yarn.lock is supposed to be committed, no?

Comment (janpio) [reply examples/javascript/script/yarn.lock]: Possible, I just noticed that all the other things were changes, and this the only addition.

**Comment by nikolasburk** [comment]
Hey @deadcoder0904, thanks a lot for this PR and apologies for the delay! We've moved all examples into the [`prisma-examples`](https://github.com/prisma/prisma-examples/tree/prisma2) repo and remove the `ts-node` dependency there 🙌

**Pull request closed without merging**